core/common/src/grpc_interceptor.rs

```diff
@@ -113,22 +97,29 @@ pub struct GrpcInterceptorService<S> {
 
 impl<S> GrpcInterceptorService<S> {
     /// Retrieve the gRPC service name and method name from a URI.
+    /// If it cannot succesfully be parsed, then an empty service name and/or method name will be returned.
```

Comment:
Consider making the return value an `Option<((String, String))>` instead. If it can't be parsed, then `None` should be returned.

Reply:
Leaving as is.